CleverTapSDK/CTPlistInfo.m

```diff
@@ -84,6 +84,13 @@ -   (instancetype)init {
         
         NSString *enableBeta = [CTPlistInfo getMetaDataForAttribute:CLTAP_BETA_LABEL];
         _beta = (enableBeta && [enableBeta isEqualToString:@"1"]);
+        
+        
+        // 1. FETCH IDFV FLAG FROM INFO.PLIST
+        NSString *disableIDFV = [CTPlistInfo getMetaDataForAttribute:CLTAP_USE_IDFV_LABEL];
+        
+        // 2. IF NOTHING FOUND, DEFAULT TO YES
+        _disableIDFV = disableIDFV ? (disableIDFV && [disableIDFV isEqualToString:@"1"]) : NO;
```

Comment:
Ternary operator usage can be avoided?

`_disableIDFV = (disableIDFV && [disableIDFV isEqualToString:@"1"]);
`